spec/vmfloaty/utils_spec.rb

```diff
@@ -31,4 +31,61 @@
       expect(Utils.generate_os_hash(host_arg)).to be_empty
     end
   end
+
+  describe '#prettyprint_hosts' do
```

Comment:
There's _some_ benefit here in specifying the desired output format, but honestly this test is going to be **super** brittle, so I don't know if it - cost vs benefit wise - is going to be very valuable. Happy to remove it, I just didn't feel like I could submit a patch without a test around the change.